v4-client-py/examples/cancel_example.py

```diff
@@ -0,0 +1,118 @@
+'''Example for trading with human readable numbers
+
+Usage: python -m examples.composite_example
+'''
+import asyncio
+import logging
+from random import randrange
+from v4_client_py.chain.aerial.wallet import LocalWallet
+from v4_client_py.clients import CompositeClient, Subaccount
+from v4_client_py.clients.constants import BECH32_PREFIX, Network
+
+from v4_client_py.clients.helpers.chain_helpers import (
+    ORDER_FLAGS_LONG_TERM,
+    ORDER_FLAGS_SHORT_TERM,
+    Order_TimeInForce,
+    OrderType, 
+    OrderSide, 
+    OrderTimeInForce, 
+    OrderExecution,
+)
+from examples.utils import loadJson
+
+from tests.constants import DYDX_TEST_MNEMONIC
+
+
+async def main() -> None:
+    wallet = LocalWallet.from_mnemonic(DYDX_TEST_MNEMONIC, BECH32_PREFIX)
+    network = Network.staging()
+    client = CompositeClient(
+        network,
+    )
+    subaccount = Subaccount(wallet, 0)
+
+    # place a long term order.
+    long_term_order_client_id = randrange(0, 2 ** 32 - 1)
+    try:
+        tx = client.place_order(
+            subaccount,
+            market='ETH-USD',
+            type=OrderType.LIMIT,
+            side=OrderSide.SELL,
+            price=40000,
+            size=0.01,
+            client_id=long_term_order_client_id,
+            time_in_force=OrderTimeInForce.GTT,
```

Comment:
Can we just remove this completely and have the client provide order flags instead?

It's really confusing that order flags are [automatically inferred from the `TimeInForce`](https://github.com/dydxprotocol/v4-clients/blob/fea76948a359186a2c387f4acf147c2507040fd2/v4-client-py/v4_client_py/clients/dydx_composite_client.py#L298), IMO client should explicitly choose what order flags to use.

Reply:
Punting this to get something functional out. This PR will be for cancels, which take in the explicit order flag, which user provides in `cancel_order`.